theme/ckeditor5-ui/globals/_reset.css

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+@custom-selector :--ck-reset-all
```

Comment:
It's the only one case of used `@custom-selector` [postcss plugin - getpostcssconfig.js](https://github.com/ckeditor/ckeditor5-dev/blob/t/321/packages/ckeditor5-dev-utils/lib/styles/getpostcssconfig.js#L17) in all stylesheets, do we really need it?

Reply:
A good point. Probably it's better to get rid of it.